slo.go

```diff
@@ -12,13 +12,26 @@ type Slos struct {
 	Slos []Slo `json:"slos"`
 }
 
+// Defines values for QueryType.
 const (
 	QueryTypeFreeform  QueryType = "freeform"
 	QueryTypeHistogram QueryType = "histogram"
 	QueryTypeRatio     QueryType = "ratio"
 	QueryTypeThreshold QueryType = "threshold"
 )
 
+// Defines values for StatusType.
+const (
```

Comment:
nit - These Status types are for the readonly section (i.e. can't modify via POST); I'd prob omit the type and just make the status in the readonly response a string type.

Up to you, there is some value in documenting the statuses that we return, but I'd prob defer that to docs or openapi specs.